test/e2e/e2e_test.go

```diff
@@ -18,11 +18,16 @@ package e2e
 
 import (
 	"testing"
+	"flag"
 
 	"github.com/kubernetes-incubator/service-catalog/test/e2e/framework"
 )
 
+var brokerImage string
```

Comment:
Done. Closed this PR to investigate the CI failure and GitHub won't let me reopen it, so I made a new PR with the change in it.